developer_manual/digging_deeper/task_processing.rst

```diff
@@ -38,6 +38,24 @@ The following built-in task types are available:
        * ``history``: ``ListOfTexts``
     * Output shape:
        * ``output``: ``Text``
+ * ``'core:text2text:chatwithtools'``: This task allows chatting with the language model with tools calling support. It is implemented by ``\OCP\TaskProcessing\TaskTypes\TextToTextChatWithTools``
+    * Input shape:
+       * ``system_prompt``: ``Text``
+       * ``input``: ``Text``
+       * ``history``: ``ListOfTexts``
+       * ``tools``: ``Text`` This should be a JSON object formatted like the one expected by the OpenAI API: https://platform.openai.com/docs/api-reference/chat/create#chat-create-tools
+    * Output shape:
+       * ``output``: ``Text``
+       * ``tool_calls``: ``Text``
+ * ``'core:contextagent:interaction'``: This task allows chatting with an agent. It is implemented by ``\OCP\TaskProcessing\TaskTypes\ContextAgentInteraction``
+    * Input shape:
+       * ``input``: ``Text``
+       * ``confirmation``: ``Number`` Whether to confirm previously requested actions. 0 to reject, 1 to confirm.
```

Comment:
`0` or **0** , `1` or **1**

or rephrased completly:

Boolean integer indicating whether to confirm previously requested actions: **0** to reject or **1** to confirm.

Reply:
:+1: done